localdev/seed/resources/seed_data.json

```diff
@@ -379,6 +379,68 @@
                 }
             ],
             "topics": [{"name": "dead courses"}]
+        },
+        {
+            "title": "Additive Manufacturing for Innovative Design and Production",
+            "subhead": "Additive Manufacturing for Innovative Design and Production.",
+            "description": "<p>Additive Manufacturing for Innovative Design and Production.</p>",
+            "readable_id": "AMxB",
```

Comment:
This isn't a correct `readable_id` although we've been using it test data.

Assuming this is a course and not a program, the readable ID should be a valid edX course key, e.g. `course-v1:SEED+AMx+R3`